cfgmgr/natmgrd.cpp

```diff
@@ -129,7 +135,11 @@ int main(int argc, char **argv)
 
         cleanupNotifier = std::make_shared<swss::NotificationProducer>(&appDb, "NAT_DB_CLEANUP_NOTIFICATION");
 
-        if (signal(SIGTERM, sigterm_handler) == SIG_ERR)
+        struct sigaction sigact;
+        sigemptyset(&sigact.sa_mask);
+        sigact.sa_flags = 0;
+        sigact.sa_handler = sigterm_handler;
```

Comment:
Duplicated code, better to extract a function.

Reply:
I simplified initialization to remove code duplication.